system/service/data_privacy.go

```diff
@@ -200,3 +203,44 @@ func (svc dataPrivacy) UpdateRequestStatus(ctx context.Context, upd *types.DataP
 
 	return r, svc.recordAction(ctx, raProps, DataPrivacyActionApprove, err)
 }
+
+func (svc dataPrivacy) FindRequestComments(ctx context.Context, filter types.DataPrivacyRequestCommentFilter) (rr types.DataPrivacyRequestCommentSet, f types.DataPrivacyRequestCommentFilter, err error) {
+	// For each fetched item, store backend will check if it is valid or not
+	filter.Check = func(req *types.DataPrivacyRequestComment) (bool, error) {
+		return true, nil
+	}
+
+	err = func() error {
+		if rr, f, err = store.SearchDataPrivacyRequestComments(ctx, svc.store, filter); err != nil {
+			return err
+		}
+
+		return nil
+	}()
```

Comment:
Are we not doing access control here?

I'd imagine we'd at least want to prevent regular users from being able to access other people's comments.

Reply:
Check it up with Chandima